src/ABLForcing/drivingForce/drivingForce.C

```diff
@@ -184,6 +184,12 @@ void Foam::drivingForce<Type>::updateComputedTimeHeightDepSource_(bool writeIter
         }
     }
 
+    // Blend source terms to constant
+    if (blendToConst_)
```

Comment:
Note that BlendToConst_ can in theory be used both with and without regression smoothing. Don't know if this is desirable or not.